devito/finite_differences/finite_difference.py

```diff
@@ -230,6 +230,9 @@ def generic_derivative(expr, dim, fd_order, deriv_order, matvec=direct, x0=None,
 
 def make_derivative(expr, dim, fd_order, deriv_order, side, matvec, x0, coefficients,
                     expand):
+    # Always expand time derivatives
```

Comment:
Can you add why? also there's a couple of leftover "I need to expand comments" from prior PR

Reply:
I already did the do string changes in previous PR

Reply:
added longer comment